README.md

```diff
@@ -124,6 +124,28 @@ Fields:
 - _Three Year Savings plan
 - _Three Year Savings plan With Azure Hybrid Benefit_
 
+### Docker
+
+#### Build the Docker image
+
+You can build a `Docker` image for the `azure-vm-pricing`:
+
+```bash
+# For Linux machines running on x86_64 or in Windows WSL
+docker build -f ./Dockerfile --platform linux/amd64 --build-arg ARCH=amd64 -t azure-vm-pricing .
```

Comment:
todo: will need to change after moving the `Dockerfile`.

Reply:
README updated, now counts with parser directory as a root directory for the parser Dockerfile.